emcc.py

```diff
@@ -3229,15 +3244,14 @@ def phase_binaryen(target, options, wasm_target):
   def preprocess_wasm2js_script():
     return read_and_preprocess(utils.path_from_root('src/wasm2js.js'), expand_macros=True)
 
-  def run_closure_compiler():
-    global final_js
-    final_js = building.closure_compiler(final_js, pretty=not minify_whitespace(),
-                                         extra_closure_args=options.closure_args)
+  if final_js:
+    if options.use_closure_compiler:
+      final_js = building.closure_compiler(final_js, pretty=not minify_whitespace(),
+                                           extra_closure_args=options.closure_args)
+    elif options.use_closure_compiler is None and settings.TRANSPILE:
```

Comment:
How about turning off TRANSPILE if the user manually sets closure to 0? Then places like this could just check TRANSPILE anod not the user's input for closure as well.